ui/app/components/nomad-icon.hbs

```diff
@@ -0,0 +1,10 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
```

Comment:
SVG code from the formerly-separate .svg file, now as a component. Used in our `<LoadingSpinner />` component.